### PR TITLE
Slicing food with Swords

### PR DESCRIPTION
**Pull request by NotSoDana**
## About the PR

Added knife utensil component to the swords. Now you can slice food, like pizza, with it.

## Why / Balance

Why we don't have this opportunity already? Slicing food with swords is a normal practice IRL, like in shows or smth like that. Of course, it'll going stain/patina on blade, but why do u need to take care bout this?

## Media

https://github.com/space-wizards/space-station-14/assets/75203942/8b8544c2-7b8c-4c11-91da-aa4a67bdc11a

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: Dezzzix
- add: Now you can slice food with swords

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/sword.yml [hunk 1]
```diff
@@ -32,6 +32,9 @@
     tags:
     - CaptainSabre
   - type: DisarmMalus
+  - type: Utensil
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Weapons/Melee/sword.yml:35]: just make BaseSword that has this and any other copy pasted components

Comment (NotSoDana) [reply Resources/Prototypes/Entities/Objects/Weapons/Melee/sword.yml:35]: sounds like a more difficult path, but.. if that actually makes sense..

**Comment by EmoGarbage404** [comment]
would it not make more sense to make slicing happening with the generic `SharpComponent`? that's what's used for butchering already.

**Comment by NotSoDana** [comment]
> would it not make more sense to make slicing happening with the generic `SharpComponent`? that's what's used for butchering already.

Hmm, sounds good, I don't thinked about that. I'll try it in BaseSword.

**Comment by NotSoDana** [comment]
Well, i've tried all i can do. Using sharp component doesn't made any impact, so will use utensil in BaseSword bruh..

**Pull request merged**